### ISSUE TITLE
autofocus does not work with buttons inside paper-dialog

### ISSUE THREAD

**Issue by 0x24a537r9** [Polymer/paper-dialog#31]
The documentation on http://www.polymer-project.org/docs/elements/paper-elements.html#paper-dialog says:

"One child element should have the autofocus attribute so that the Enter key will automatically take action. This is especially important for screen reader environments."

Even when one button has the autofocus attribute as in the demo (http://www.polymer-project.org/components/paper-dialog/demo.html) the Enter key does nothing. I'm not sure whether the issue is with autofocus or the dialog, but something is broken. This is a real problem from an accessibility standpoint, and an annoyance to other users as well.
